driver/Configuration.ml

```diff
@@ -10,6 +10,7 @@
 (*                                                                     *)
 (* *********************************************************************)
 
+open Filename
```

Comment:
Cosmetic: I prefer not to open Filename and qualify explicitly.  E.g. "Filename.concat" is more readable than "concat" (which could be some other kind of concatenation operation).